team-agreements/working-agreements/readme.md

```diff
@@ -1,15 +1,63 @@
 # Sections of a working agreement
 
+A working agreement is a document or a set of documents that describe how we work together as a team and what our expectations and principles are.
+
+The working agreement created by the team at the beginning of the project, and is stored in the repository so that it is readily available for everyone working on the project.
+
+The following are examples of sections and points that can be part of a working agreement but each team should compose their own, and adjust times etc. to fit their team needs.
+
+## General
+
+- We work as one team towards a common goal and clear scope
+- We make sure everyones voice is heard, listened to
+- We show all team members equal respect
+- We pair-program as often as possible
+- We make sure to spread our expertise and skills in the team, so no single person is relied on for one skill
+- All times below are listed in CET
+
+## Communication
+
+- We communicate all information relevant to the team through the Project Teams channel
+- We add all research results, design documents and other technical documentation to the project repository through PRs
+
+## Work life balance
+
+- Our office hour, when we can expect to collaborate via Microsoft Teams, phone or face-to-face are Monday to Friday 10AM-5PM
+- We are not expected to answer emails past 6PM, on weekends on when we are on holidays or vacation.
+- We work in different time zones and respsect this, especially when setting up recurring meetings.
```

Comment:
Not all projects are across different time zones: If we work in different time zones?

Reply:
See comments earlier - this is a set of samples, times, communication channels and applicable sections should be added, removed or adjusted with the team